jwql/website/apps/jwql/data_containers.py

```diff
@@ -29,19 +29,28 @@
 
 from astropy.io import fits
 from astropy.time import Time
-from astroquery.mast import Mast
+
 import numpy as np
 
+# astroquery.mast import that depends on value of auth_mast
```

Comment:
@Johannes-Sahlmann I think this is fine.  It's not a problem to stray from conventions as long as there is a good reason and it is well documented, as is the case here.